src/mnode/mnode-pastel.h

```diff
@@ -529,6 +529,72 @@ class CArtTradeTicket : public CPastelTicket
     std::unique_ptr<CPastelTicket> FindArtRegTicket() const;
 };
 
+/*
+  "ticket": {
+    "type": "royalty",
+    "version": "",
+    "pastelID": "",     //pastelID of the old (current at moment of creation) royalty recipient
+    "new_pastelID": "", //pastelID of the new royalty recipient
+    "art_txid": "",     //txid of the art for royalty payments
+    "signature": ""
+  }
+*/
+
+class CArtRoyaltyTicket : public CPastelTicket {
+public:
+  std::string pastelID;    //pastelID of the old (current at moment of creation) royalty recipient
```

Comment:
We can also agree on naming class member variables.
If we use "m_XXX" prefix (at least for all new code) - it would be much easier to read all class member functions code - to distinguish between local variables, global variables and class member variables.
this one could be std::string m_sPastelID;

Reply:
I agree with you )

Reply:
wiki says it is also part of Hungarian notation :) I'm not saying we should blindly follow that, it's only about code readability. Also, it's very convenient - when you start typing m_... Intellisense will show you all class member variables for your scope, not local vars or global vars.